wiki/mapping/basic-mapping.md

```diff
@@ -47,47 +54,337 @@ As a *very general* rule of thumb for new mappers, blocks should be distributed
+
+#### Easy to Miss Patterns
+| Illustration | Editor View | Explanation |
+|:----:|:----:|:---- |
+| ![Illustration of overlapping hitboxes](./images/overlapping-hitboxes.png) | ![Editor view of overlapping hitboxes](./images/overlapping-hitboxes-alt.png) | Generally, try to avoid patterns with overlapping hitboxes, as this makes the pattern difficult to hit consistently. The hitboxes are larger than the note, and most players rarely hit the actual note when they swing, making it easy to hit the smaller hitbox within notes that detects bad hits on patterns like this. See [Hitbox Abuse](./basic-mapping#hitbox-abuse) for more info.  |
+
+#### Arm Tangles
+| Illustration | Editor View | Explanation |
+|:----:|:----:|:---- |
+| ![Illustration of arm tangles](./images/arm-tangle.png) | ![Editor view of arm tangles](./images/arm-tangle-alt.png) | Crossing hands is usually fine, but crossing hands and going in opposite directions vertically causes the player’s arms to hit each other. Reverse these colors and you’re good to go. This is only one example of an arm tangle. Always be aware of where you're leaving your player's arms and how they are going to escape from that position to hit the next pattern. |
+
+## Gauging Difficulty & Down-Mapping
+Many mappers tend to map the difficulty level at which they play but there are plenty of benefits to mapping lower difficulties:
+
+* Full spreads widen your audience leading to more downloads
+* They look more professional when cruising for new maps... like official game maps
+
+The actual difficulty level of a song is a combination of note density, pattern complexity, and note speed and spawn point. When down-mapping it’s important to understand what skills players at different levels have and how to use that information to develop a sense of progression in your levels. 
+
 ### Relative Difficulty
+Maps should always have a progression of complexity from Easy through Expert+. The actual difficulty range of these levels in terms of note density has been creeping up since the game first came out. 
+
+While it’s best to stick within conventional ranges to start, as long as your levels have a clear progression of difficulty they don’t need to fit neatly into a box. If you’re only doing one difficulty you should stay within the expected range. Your map’s difficulty should fit the song - a 200 BPM Camellia song is going to be more challenging than a 120 BPM pop song.
+<p align="center">
+  <img src="./images/ost-extras-nps.png" alt="a table of NPS values from official content">
+</p>
+
+The above OST Extras examples show how tracks like “What The Cat!?” are quite dense at the highest difficulty but the easy is also more dense than other easy levels. In each of these cases there is still a progression from Easy to ExpertPlus
```

Comment:
Despite the file name and alt text, this table isn't labeled when viewed in the wiki, so it's unclear that it's showing the NPS (Notes Per Second).
![image](https://user-images.githubusercontent.com/27714637/71597229-30aaa580-2af7-11ea-9f4b-5c5f5b423bf4.png)
Consider changing the text here to something like 
`The above table shows the average NPS (Notes Per Second) of the OST Extras maps. You can see how tracks like...`

I see this is explained later in the `note-density` section. Consider adding a link to that section here as well